plugins/info/info_impl.cpp

```diff
@@ -34,6 +34,16 @@ void InfoImpl::deinit()
     _parent->unregister_all_mavlink_message_handlers(this);
 }
 
+void InfoImpl::enable()
+{
+
```

Comment:
Same thing here: either `void InfoImpl::enable() {}`, or at least remove the empty line :-).